language_tags/tags.py

```diff
@@ -81,7 +81,7 @@ def search(description, all=False):
         """
         Gets a list of :class:`language_tags.Subtag.Subtag` objects where the description matches.
 
-        :param description: a string or compiled regular expression. For example: ``search(re.compile('\d{4}'))`` if the
+        :param description: a string or compiled regular expression. For example: ``search(re.compile(r'\d{4}'))`` if the
```

Comment:
The deprecation warning is still triggered.
```
language_tags/tags.py:81

  /home/travis/build/OnroerendErfgoed/language-tags/language_tags/tags.py:81: DeprecationWarning: invalid escape sequence \d
```
It is possible to use a raw multiline string so that might be the solution
![afbeelding](https://user-images.githubusercontent.com/1926276/114979909-f00da180-9e8b-11eb-8d00-08e4d71d3016.png)